content/_code-samples/quickstart/js/8.escrow.html

```diff
@@ -0,0 +1,305 @@
+<html>
```

Comment:
Best practice is to add a DOCTYPE so browsers don't use "quirks mode" to render the document.

```suggestion
<!DOCTYPE html>
<html>
```